### PR TITLE
TE-356 little fixes

### PR DESCRIPTION
**Pull request by marcdavi-es**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-X)

### What **one** thing does this PR do?
- Adds logo with 'Lodgify UI' title
- Fixes heading 1 styles

**Before**

screen shot 2018-05-01 at 17 50 44

screen shot 2018-05-01 at 17 51 00

**After**

screen shot 2018-05-01 at 17 50 24

screen shot 2018-05-01 at 17 50 28

### CODE REVIEW

File: styleguide/styleguide-components/LogoRenderer/component.js [hunk 1]
```diff
@@ -3,8 +3,10 @@ import React from 'react';
 import * as logo from './logo.png';
 
 export const component = ({ hasNoMargin }) => (
-  <img
-    src={logo.default}
-    style={{ height: 100, marginLeft: hasNoMargin ? 0 : 20 }}
-  />
+  <a href="/">
+    <img
```
Comment (Jonnygwi) [inline styleguide/styleguide-components/LogoRenderer/component.js]: piff

File: styleguide/styleguide-components/MobilePlaceholder/component.js [hunk 2]
```diff
@@ -67,7 +67,7 @@ class Markup extends React.Component {
       <div className={classes.screen} onScroll={console.log}>
         <div>
           <LogoRenderer hasNoMargin />
-          <h1 className={classes.heading}>Yo</h1>
+          <p className={classes.firstParagraph}>Yo.</p>
```
Comment (Jonnygwi) [inline styleguide/styleguide-components/MobilePlaceholder/component.js]: 👌

File: styleguide/styleguide-components/MobilePlaceholder/component.js [hunk 3]
```diff
@@ -22,9 +22,9 @@ const styles = ({ borderRadius, color, mq, space }) => ({
       visibility: 'visible',
     },
   },
-  heading: {
+  firstParagraph: {
     isolate: false,
-    lineHeight: '1em',
+    marginTop: space[4],
```
Comment (Jonnygwi) [inline styleguide/styleguide-components/MobilePlaceholder/component.js]: 👨‍🚀

File: styleguide/styleguide-components/LogoRenderer/component.js [hunk 4]
```diff
@@ -3,8 +3,10 @@ import React from 'react';
...
+      src={logo.default}
+      style={{ display: 'block', margin: `0${hasNoMargin ? '' : ' auto'}`, width: '110px', }}
+    />
```
Comment (davidpelayo) [inline styleguide/styleguide-components/LogoRenderer/component.js]: There is an `alt` attribute missing here.

**Pull request merged**